InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveConstants.java

```diff
@@ -93,6 +93,9 @@ public static double ticksP100ToIps(int ticksP100)
     public static final double MAX_LIN_ACCELERATION_IPSPS = 999999999;
 
 
+    public static final int FILTER_LENGTH = 25;
```

Comment:
since it's not for the drive subsystem, you should make a ShooterConstants instead and use those, but this line of code is correct! so just cut-paste it into a new ShooterConstants in the shooter subsystem folder